requirements/edx/base.txt

```diff
@@ -52,7 +52,7 @@ edx-lint==0.4.3
 astroid==1.3.8
 edx-django-oauth2-provider==1.1.4
 edx-django-sites-extensions==2.1.1
-edx-enterprise==0.26.3
+edx-enterprise==0.27.0
```

Comment:
@asadiqbal08 there is no release named `0.27.1`, If I create it now then it will also contain other PRs that I do not know how to test at this point.